### DISCUSSION TITLE
Validate nested dependencies?

### DISCUSSION THREAD

**Discussion by Jertimmer** [renovatebot/renovate#10387] [Request Help]
I've got a project where we use package A version 1.0 which has an exact dependency on package B version 1.5. We also use package B directly in our project, so when we run Renovate, it creates a MR to update B to 2.0, which fails because 2.0 is not supported by package A. 
Is there any way we can solve this elegantly in the config by having Renovate validate the dependencies of our packages?

**Comment by viceice** [comment]
Nope, that's not supported by renovate. You can suppress major updates easily by using dependency dashboard approval for major updates. The dashboard also shows an overview of all renovate updates.

**Reply by Jertimmer** [reply]
thanks for the quick reply.
